### PR TITLE
add caller_id_numbers field to kt_rates import

### PR DESCRIPTION
**Pull request by onnet**
in continuation of #4608

### CODE REVIEW

File: applications/tasks/src/modules/kt_rates.erl [hunk 1]
```diff
@@ -439,3 +440,13 @@ generate_weight(?NE_BINARY = Prefix, UnitCost, UnitIntCost) ->
 
     Weight = (byte_size(Prefix) * 10) - trunc(CostToUse * 100),
     kzd_rate:constrain_weight(Weight).
+
+-spec maybe_generate_caller_id_numbers(kzd_rate:doc()) -> kz_term:ne_binaries()|'undefined'.
+maybe_generate_caller_id_numbers(RateJObj) ->
+    maybe_generate_caller_id_numbers(RateJObj, kz_json:get_value(<<"caller_id_numbers">>, RateJObj)).
+
+-spec maybe_generate_caller_id_numbers(kzd_rate:doc(), kz_term:ne_binary()) -> kz_term:ne_binaries()|'undefined'.
```
Comment (jamesaimonetti) [inline applications/tasks/src/modules/kt_rates.erl]: Return type should be `kz_term:api_ne_binaries()`.

File: core/kazoo_documents/src/kzd_rate.erl [hunk 2]
```diff
@@ -277,6 +277,10 @@ caller_id_numbers(Rate) ->
 caller_id_numbers(Rate, Default) ->
     kz_json:get_list_value(<<"caller_id_numbers">>, Rate, Default).
 
+-spec set_caller_id_numbers(doc(), kz_term:ne_binaries()) -> doc().
+set_caller_id_numbers(Rate, CID_Numbers) when is_list(CID_Numbers) ->
```
Comment (jamesaimonetti) [inline core/kazoo_documents/src/kzd_rate.erl]: You should add the field to the `rates.json` schema.

**Comment by k-anderson** [comment]
Howdy, just a note that #4608 was merged into 4.2 prior to the freeze.  If this is required for that PR to work properly please make sure there is a backport :)

Thanks!

**Comment by lazedo** [comment]
@onnet does this need work after #4645 ?

**Comment by onnet** [comment]
@lazedo it appears to be partially already in. Will rebase and check what's going on

**Comment by jamesaimonetti** [comment]
Merged

**Pull request closed without merging**